app/js/FileController.js

```diff
@@ -61,7 +61,8 @@ function getFile(req, res) {
     }
 
     logger.log({ key, bucket, format, style }, 'sending file to response')
-    pipeline(fileStream, res)
+
+    pipeline(fileStream, res, next)
```

Comment:
maybe add the commit message as a comment here, it wasn't immediately obvious that `next` was being used as the callback.